types/knockout/index.d.ts

```diff
@@ -1,10 +1,7 @@
 // Type definitions for Knockout v3.4.0
 // Project: http://knockoutjs.com
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>
-//                 Igor Oleinikov <https://github.com/Igorbek/>
-//                 Clément Bourgeois <https://github.com/moonpyk/>
-//                 Matt Brooks <https://github.com/EnableSoftware>
-//                 Benjamin Eckardt <https://github.com/BenjaminEckardt>
+// Definitions by: Boris Yankov <https://github.com/borisyankov/>, Igor Oleinikov <https://github.com/Igorbek/>, Clément Bourgeois <https://github.com/moonpyk/>, Matt Brooks <https://github.com/EnableSoftware>, Benjamin Eckardt <https://github.com/BenjaminEckardt>, Mathias Lorenzen <https://github.com/ffMathy>
+// TypeScript Version: 2.3
```

Comment:
This should go *below* the next line.
Also, for lists this long the multi-line version is preferable.

Reply:
I don't understand this. Which line goes under which? I can't make it work. I tried numerous combinations.

Is this documented somewhere? Can you help me tell how it should look instead?